app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>wifiextender@yandex.com</email>
+		<name>Aaron Caffrey</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">su8/pinky-bar</remote-id>
+	</upstream>
+	<use>
+		<flag name="alsa">Get the sound volume level using ALSA.</flag>
+		<flag name="colors">Colorize the output data.</flag>
+		<flag name="drivetemp">Read the drive temperature from S.M.A.R.T.</flag>
+		<flag name="drivetemp-light">Read the drive temperature from S.M.A.R.T
+			light version.</flag>
```

Comment:
As a potential user, I would like to know what the difference between the two is and why I should choose one over the other.